.github/README.md

```diff
@@ -107,6 +107,11 @@ composer update wpackagist-plugin/hello-dolly
 **Please note:** If the resulting diff from a Composer workflow goes beyond `composer.json` and `composer.lock`, please
 double-check what is being changed. It is likely that something unexpected has happened; proceed with caution.
 
+**Please note:** Our linting rules expect that you will manually exclude third party plugins in our `phpcs.xml` config.
+For example: `<exclude-pattern>web/app/plugins/FOO</exclude-pattern>`. We don't exclude _all_ plugins as we want to
```

Comment:
Using FOO as an example is fine, obviously, but when I split the readme into component pages in the next sprint I'm likely to use a standard example through this section (either all FOO, or all Hello Dolly).